pandas/io/pickle.py

```diff
@@ -72,6 +93,28 @@ def read_pickle(path, compression='infer'):
     Returns
     -------
     unpickled : type of object stored in file
```

Comment:
add a See Also and ref DataFrame.to_pickle, pd.read_hdf, pd.read_sql, pd.read_parquet

Reply:
added.